docpage/docs/Component.md

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 5
+---
+# Component
+
+
+Component subsection is use in all Elements like Kafka, OSCAR or a Generic one. This subsection changes key parts of the workflow's operation. However, it is necessary to know the names of the Process that make up the ProcessGroup. By indicating the name of the Process, you can change the execution time and the node on which it runs (ALL or PRIMARY).
```

Comment:
The component subsection is used in all Elements like Kafka, OSCAR, and the Generic one. A component alters the workflow's operation by employing Apache NiFi Processors. The Processor's name, execution time and the node on which it runs (ALL or PRIMARY) must be indicated.